zstacklib/zstacklib/network/bridge/operations.py

```
@@ -0,0 +1,233 @@
+                    return line.split('=')[1].strip()
+    except (OSError, IOError):
+        pass
+    return None
+
+
+def create_bridge(
+    name: str,
+    interface: str | None = None,
+    stp: bool = False,
+    forward_delay: int = 0,
+    move_route: bool = True
```

Ruff (0.15.2)

[warning] Unused function argument: `move_route`

(ARG001)